targets/REV3-LVSS/main.cpp

```diff
@@ -48,13 +48,46 @@ int main() {
     log::LOGGER.setUART(&uart);
     log::LOGGER.setLogLevel(log::Logger::LogLevel::INFO);
 
-    IO::GPIO& lvssEn0 = IO::getGPIO<IO::Pin::PA_10>(IO::GPIO::Direction::OUTPUT);
+    IO::GPIO& lvssPowerSwitch0Enable0 = IO::getGPIO<IO::Pin::PC_14>(IO::GPIO::Direction::OUTPUT);// hib
```

Comment:
You should probably name these values somewhere (i.e. in LVSS.hpp) instead of just keeping them as magic values. Then you don't have to worry about commenting what they are- they'll be named!